### PR TITLE
Build: React, add "... otherProps"

### PR DESCRIPTION
**Pull request by adambbecker**
Adds support for other React props to be passed to the icons such as title, style, etc.

### CODE REVIEW

**Comment by gwwar** [comment]
Looks like we have a few other changes we'll like to queue up for another release. I'm going to go ahead and merge this @adambbecker

File: Gruntfile.js [hunk 1]
```diff
@@ -317,7 +317,7 @@ module.exports = function( grunt ) {
 
 			// Add className, height, and width to the svg element
 			fileContent = fileContent.slice( 0, 4 ) +
-						' className={ iconClass } height={ size } width={ size } onClick={ onClick }' +
+						' className={ iconClass } height={ size } width={ size } onClick={ onClick } { ...otherProps }' +
```
Comment (gwwar) [inline Gruntfile.js]: @adambbecker looks like we can override classname, so we'll want to add special handling for that prop

cc @folletto

File: sources/react/index-header.jsx [hunk 2]
```diff
@@ -150,5 +150,5 @@ export default class Gridicon extends PureComponent {
 
 		switch ( icon ) {
 			default:
-				svg = <svg height={ size } width={ size } />;
+				svg = <svg height={ size } width={ size } { ...otherProps } />;
```
Comment (gwwar) [inline sources/react/index-header.jsx]: Ah reading though again we'll want to omit className since we do have special handling for it already

**Comment by adambbecker** [comment]
@gwwar Ah, I see. Yeah it looks like #235 (thank you!) provides a fix for this. Is that the case or should we reopen this?

**Comment by gwwar** [comment]
Yes, @adambbecker #235 should should fix this. 😄

**Pull request merged**